gdal/frmts/mrf/JPEG_band.cpp

```diff
@@ -679,7 +679,7 @@ JPEG_Band::JPEG_Band( MRFDataset *pDS, const ILImage &image,
     }
 
     if( GDT_Byte == image.dt )
-        codec.optimize = GetOptlist().FetchBoolean("OPTIMIZE", FALSE) != FALSE;
+        codec.optimize = (true == GetOptlist().FetchBoolean("OPTIMIZE", FALSE));
```

Comment:
can you explain why your change changes behaviour ? To me, the version before and after this change are equivalent.